Update admission_controller.md (#15681)
DataDog/documentation: content/en/tracing/trace_collection/admission_controller.md

{{< beta-callout url="#" btn_hidden="true">}}
  Tracing library injection using Admission Controller is in beta. 
{{< /beta-callout >}}

> In most environments, configuring your application to send traces to Datadog involves two steps:

Reviewer: We can be specific in there we are talking about Kubernetes environment so the steps should be clear

- Configuring the Datadog Agent for APM.
- Adding the Datadog Tracing Library to your code.

> For your Kubernetes applications that send traces to Datadog by the Cluster Agent (version 7.39 and higher for Java and JavaScript, version 7.40 and higher for Python), you can configure the Admission Controller to inject APM Java and JavaScript tracing libraries automatically.

Reviewer: Is 7.40 released already?

> By automating the injection of tracer libraries through the admission controller, you don't have to change your application images, helping you get up and running with APM quickly.

Reviewer 2: ```suggestion
By automating the injection of tracer libraries through the Admission Controller, you don't have to change your application images, helping you get up and running with APM quickly.
```

> After you install the Datadog Agent, and then the [Cluster Agent][1], do one of the following:

Reviewer: Datadog agent and cluster agent are the same setup

Valid `<language>` values are:
- `java`
- `js`
- `python`

For example to inject the Java tracer v1.0.0:

```yaml
annotations:

>     admission.datadoghq.com/java-lib.version: "1.0.0"

Reviewer: I don't think v1.0.0 is available for java in this mode ( at least for [now](https://hub.docker.com/repository/registry-1.docker.io/datadog/dd-lib-java-init/tags?page=1&ordering=last_updated)).

```

**Note**: Use caution specifying `latest` as major library releases can introduce breaking changes.

Although it's an uncommon scenario, you can add multiple `<language>-lib.version` annotations to inject multiple language tracers into one container.

> For example to inject the Java tracer v1.0.0, Python tracer v1.0.0, and Node tracer v3.0.0:

Reviewer: Can we add a where to find the versions available by language?

```yaml
annotations:
    admission.datadoghq.com/java-lib.version: "1.0.0"
    admission.datadoghq.com/python-lib.version: "1.0.0"
    admission.datadoghq.com/js-lib.version: "3.0.0"
```

You can also inject the latest version of a tracer library for the Java, Python and Node language via the following annotation:
```yaml
annotations:
    admission.datadoghq.com/java-lib.version: "latest"
    admission.datadoghq.com/python-lib.version: "latest"
    admission.datadoghq.com/js-lib.version: "latest"
```

> Note that the latest keyword should be used with caution, as it may risk potential errors due to major version differences from your newly injected applications versus your existing applications that have already been deployed. 

Reviewer 2: ```suggestion
**Note**: Use caution specifying `latest`, as it may raise errors because of major version differences between your newly injected applications and your existing applications that have already been deployed. 
```

| `false`          | `admission.datadoghq.com/enabled=false` | No        |

[1]: /containers/cluster_agent/setup/?tab=helm
[2]: /containers/cluster_agent/admission_controller/

> [3]: /k8-injection-doc-updates/containers/kubernetes/installation/?tab=helm

Reviewer 3: ```suggestion
[3]: /containers/kubernetes/installation/?tab=helm
```
